test_driver/setup/global_consts.dart

```diff
@@ -103,6 +103,10 @@ final typeSomethingComposePlaceholder = L.getKey(L.typeSomething);
 
 const helloWorld = 'Hello world';
 
+final delete = "Delete";
+
+final scrollDuration = Duration(milliseconds: 1000);
```

Comment:
As this value is applied at multiple places within the tests, please check if it is necessary to add it as parameter to multiple methods (as it's currently done) or if we can use it directly within the methods. If it's possible that the different methods, using this value, will use different values in the future, the method parameter could be suitable, otherwise the constant should be used directly.

Reply:
You are right.
It is used more than one time in helper methods.dart .
So I thought it could been possible to use it otherwise.
So I am going to define it only in the helper_method.dart.